radixdlt-java/src/main/java/com/radixdlt/client/atommodel/Fungible.java

```diff
@@ -0,0 +1,16 @@
+package com.radixdlt.client.atommodel;
+
+import org.radix.utils.UInt256;
+
+/**
+ * Temporary interface for representing an amount of interchangeable Particles.
+ */
+public interface Fungible {
+	UInt256 getAmount();
```

Comment:
Should really have some Javadoc for each of these.

Reply:
Usually, I didn't include any in this case because I assumed it would be changed in the next version anyway.